cms/djangoapps/contentstore/views/videos.py

```diff
@@ -338,11 +340,13 @@ def transcript_preferences_handler(request, course_key_string):
 
     Returns: valid json response or 400 with error message
     """
+    transcript_preferences = None
     data = request.json
-    provider = data.get('provider', '')
+    provider = data.get('provider', None)
 
-    # TODO: if provider == '': delete course preferences
-    # i.e call delete api end point like delete_transcript_preferences(course_key_string)
+    if not provider:
+        remove_transcript_preferences(course_key_string)
+        return JsonResponse({'transcript_preferences': transcript_preferences}, status=200)
```

Comment:
`transcript_preferences` is always `None`?

Reply:
In case of of no provider selected

Reply:
got it.